src/pages/patterns/errors.mdx

```diff
@@ -0,0 +1,311 @@
+---
+title: Error handling
+description: >
+  Error messages let users know that a problem has occurred and what to do about it.
+---
+
+Examples of errors include:
+
+- The system has failed to load or perform an action
+- The user has made an incorrect action or input
+
+Error messages are best surfaced using these components:
+
+- Global Alerts
+- Alerts
+- Notifications
+- Validation messages (in Forms)
+
+## Global Alerts
+
+Use the Global Alert to communicate system level errors that interrupt a user’s task.
+
+### Global alert anatomy
+
+- Always include a title to explain what has went wrong and how to fix it
+- Provide a clear action to address the issue
+
+![Global Alert anatomy](figma:patterns-error-global-alert-anatomy.png)
+
+1. Title
+2. Message
+3. Button
```

Comment:
```suggestion
1. Title
1. Message
1. Button
```

[nit] the suggested markdown follows Garden's preference for ordered lists. Consider applying throughout.